### ISSUE TITLE
Documentation: Updating Widevine troubleshooting

### ISSUE THREAD

**Issue by TeenCorn** [qutebrowser/qutebrowser#5112]
Instead of

`For other distributions, it should be possible to obtain the needed widevine files and store them in the correct places, but the details differ wildly between various Qt versions.`

Make it a list of Qtwebengine versions and the corresponding path(s) that version of webengine looks at for widevine. So,
 ```
{Qtwebengine version} - { path1,
                           path2,
                           ... }
```

Example:
```
5.13.2 - { /opt/google/chrome/libwidevinecdm.so,
           /usr/lib/chromium/libwidevinecdm.so,
           /usr/lib/chromium-browser/libwidevinecdm.so,
           /usr/lib64/chromium/libwidevinecdm.so }

{Other version} - { ... }
```
Place to look at for paths qtwebengine look a [here](https://github.com/qt/qtwebengine/blob/b97f0eb87b6e6a6726b805c9de2159c6ad983f7d/src/core/content_client_qt.cpp). Search for Q_OS_LINUX

So that the user can installed chrome and put libwidevinecdm.so in that path so they can watch netflix and other drm protected media.

**Comment by The-Compiler** [comment]
It's not as easy as you make it sound - the paths are the smallest problem (and they are mostly consistent across Qt versions).

However, depending on the underlying Chromium version:

- You need a matching Widevine version, newer versions won't work.
- There is just a widevine file, or there is that plus a `widevinecdmadapter.so` file
- Those files are shipped as part of the Chromium project, or Widevine

So the most important thing to find out is what version is supported, what files are needed, and what's a trustworthy place to get them. I'm not going to do that work for all supported Qt versions - if you (or someone else) knows about a known-good configuration, I'd be happy about a pull request though!

**Comment by TeenCorn** [comment]
Yeah, I was not expecting you to be the one to document all this lol. I just wanted anybody that knew how to get widevine working for their particular version of qutebrowser to submit a pull request to update the documentation like you said.

I spent too much time trying to fix this problem after I uninstalled chrome and realized that netflix no longer worked :sweat_smile:

But I'll see what I can do to document how I fixed my case.
